ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/js/datepickerview.js

```diff
@@ -0,0 +1,56 @@
+/*******************************************************************************
+ * Copyright 2022 Adobe
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+(function() {
+
+    class DatePicker extends FormView.FormField {
+
+        static NS = "cmp";
+        static IS = "adaptiveFormDatePicker";
+        static selectors  = {
+            self: "[data-" + this.NS + '-is="' + this.IS + '"]',
+            dateElement: "input[type=date]"
+        };
+
+        constructor(params) {
+            super(params);
+        }
+
+        getClass() {
+            return DatePicker.IS;
+        }
+
+        getTagName() {
+            return "input";
+        }
+
+        setValue(value) {
+            let dateInput = this.element.querySelector(DatePicker.selectors.dateElement);
+            dateInput.value = value;
+        }
+    }
+
+    function onFormContainerInitialised(e) {
+        console.log("FormContainerInitialised Received", e.detail);
+        let formContainer =  e.detail;
```

Comment:
Please check if this path and path from sightly is same (you need to re-use data-cmp-adaptiveformcontainer-path)

Reply:
this will the container itself, not the path.